repository.owl

```diff
@@ -0,0 +1,29 @@
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
```

Comment:
Would an `application/owl+xml` serialization make sense for better HTML/XSLT rendering?
https://www.w3.org/TR/owl2-xml-serialization/